source/requirements.txt

```diff
@@ -9,5 +9,6 @@ myst_nb
 torch
 torchvision
 kornia
+imageio
```

Comment:
imageio-ffmpeg ?

Reply:
okay - then what I would do is to remove from running to avoid broken things
https://github.com/kornia/tutorials/blob/master/source/conf.py#L54